providers/implementations/kdfs/scrypt.c

```diff
@@ -56,7 +56,7 @@ typedef struct {
 
 static void kdf_scrypt_init(KDF_SCRYPT *ctx);
 
-static void *kdf_scrypt_new(void *provctx)
+static void *kdf_scrypt_new_inner(OSSL_LIB_CTX *libctx)
```

Comment:
Is this caught from a different PR?  It doesn't seem releated.
But it is a fix.

Reply:
It should be rebased since #17873 has been merged

Reply:
Thanks.  No merge conflict is showing which is a bit odd.